tests/e2e/e2e.go

```diff
@@ -259,3 +287,32 @@ func CheckBootstrapIsPossible(network testnet.Network) {
 	node := AddEphemeralNode(network, testnet.FlagsMap{})
 	WaitForHealthy(node)
 }
+
+// Start a local test-managed network with the provided avalanchego binary.
+func StartLocalNetwork(avalancheGoExecPath string, networkDir string) *local.LocalNetwork {
+	require := require.New(ginkgo.GinkgoT())
+
+	tests.Outf("{{magenta}}Starting network with %q{{/}}\n", avalancheGoExecPath)
+
+	network, err := local.StartNetwork(
+		DefaultContext(),
+		ginkgo.GinkgoWriter,
+		networkDir,
+		&local.LocalNetwork{
+			LocalConfig: local.LocalConfig{
+				ExecPath: avalancheGoExecPath,
+			},
+		},
+		testnet.DefaultNodeCount,
+		testnet.DefaultFundedKeyCount,
+	)
+	require.NoError(err)
+	ginkgo.DeferCleanup(func() {
+		tests.Outf("Shutting down network\n")
+		require.NoError(network.Stop())
+	})
```

Comment:
feature request: disable this step with flag or ENV

Reply:
having it optional by ENV seems non intrusive and useful for debug, just for future considerations :)